forms-shared/src/definitions/emailFormTypes.ts

```diff
@@ -0,0 +1,12 @@
+export enum MailgunTemplateEnum {
+  NASES_SENT = 'NASES_SENT',
+  GINIS_DELIVERED = 'GINIS_DELIVERED',
+  NASES_GINIS_IN_PROGRESS = 'NASES_GINIS_IN_PROGRESS',
+  GINIS_SUCCESS = 'GINIS_SUCCESS',
+  GINIS_DENIED = 'GINIS_DENIED',
+  ATTACHMENT_VIRUS = 'ATTACHMENT_VIRUS',
+  OLO_SEND_FORM = 'OLO_SEND_FORM',
+  OLO_DELIVERED_SUCCESS = 'OLO_DELIVERED_SUCCESS',
```

Comment:
OLO_DELIVERED_SUCCESS = TSB_SENT_SUCCESS?
OLO_SEND_FORM = TSB_NEW_SUBMISSION?

Can we unify the naming?